.buildkite/scripts/make_image.sh

```diff
@@ -86,12 +86,17 @@ if [[ "$BUILD_TYPE" == "miner" ]]; then
         echo "Miner tag update detected; Updating ${UPDATE_TAG} on ${REGISTRY_HOST}..."
 
         UPDATE_TAG="${UPDATE_TAG}-${IMAGE_ARCH}"
-        DOCKER_NAME=$(echo "$DOCKER_NAME" | sed -e 's/_GA//' -e 's/_alpha//' -e 's/_beta//')
+        SEMVER_TAG=$(echo "$DOCKER_NAME" | sed -e 's/_GA//' -e 's/_alpha//' -e 's/_beta//')
 
-        docker pull "$MINER_REGISTRY_NAME:$DOCKER_NAME"
-        docker tag "$MINER_REGISTRY_NAME:$DOCKER_NAME" "$MINER_REGISTRY_NAME:$UPDATE_TAG"
+        docker pull "$MINER_REGISTRY_NAME:$SEMVER_TAG"
+        docker tag "$MINER_REGISTRY_NAME:$SEMVER_TAG" "$MINER_REGISTRY_NAME:$UPDATE_TAG"
         docker push "$MINER_REGISTRY_NAME:$UPDATE_TAG"
 
+        if [["$VERSION_TAG" =~ _GA$]]; then
```

Comment:
not sure this is explicitly necessary since this should only ever have a single value in any given build but to prevent any conflicts/mismatch between version shell variables this could also be `if [["$DOCKER_NAME" =~ _GA$]]; then`